Makefile

```diff
@@ -42,40 +38,42 @@ OTELCOL_BINARY=google-cloudops-opentelemetry-collector_$(GOOS)_$(GOARCH)$(EXTENS
 
 .PHONY: build
 build:
-	go build -o ./bin/$(OTELCOL_BINARY) ./cmd/otelopscol
+	go build -o ./bin/$(OTELCOL_BINARY)_$(GOOS)_$(GOARCH)$(EXTENSION) ./cmd/otelopscol
 
 # googet (Windows)
-.PHONY: build-googet
-build-googet:
-	GOOS=windows
-	build package-googet
-
-.PHONY: package-googet
-package-googet: SHELL:=/bin/bash
-package-googet:
-	GOOS=windows
+
+.PHONY: build-goo
+build-goo: export GOOS=windows
+build-goo: export EXTENSION=.exe
+build-goo: build package-goo
+
+.PHONY: package-goo
+package-goo: export GOOS=windows
+package-goo: export EXTENSION=.exe
+package-goo: SHELL:=/bin/bash
+package-goo:
+	mkdir -p dist
 	# goopack doesn't support variable replacement or command line args so just use envsubst
 	goopack -output_dir ./dist <(envsubst < ./.build/googet/google-cloudops-opentelemetry-collector.goospec)
+	chmod -R 777 ./dist/
```

Comment:
Need this because when running these commands in a docker container, everything is created under the "root" user. On Kokoro, the build agent runs as the kokoro user and can't access the generated files. Not sure if there is a better solution but this works well enough for now.